QUALITY.md

```diff
@@ -0,0 +1,193 @@
+A full copy of the license can be found [here](LICENSE).
+
+### Copyright Statements [3.iv]
+
+**eProsima Micro CDR** copyright holder provide a statement of copyright in each source file.
+
+## Testing [4]
+
+### Feature Testing [4.i]
+
+**eProsima Micro CDR** provides tests which simulate typical usage, and they are located in the [`test` directory](test).
+New features are required to have tests before being added as stated in [CONTRIBUTING](CONTRIBUTING.md).
+Current nightly results can be found here:
+
+* Linux [![Linux ci](http://jenkins.eprosima.com:8080/view/Nightly/job/Micro-CDR%20Nightly%20Master%20Linux/badge/icon?subject=%20%20%20Linux%20CI%20)](http://jenkins.eprosima.com:8080/view/Micro%20XRCE/job/Micro-CDR%20Nightly%20Master%20Linux/)
+* Windows [![Windows ci](http://jenkins.eprosima.com:8080/job/Micro-CDR%20Nightly%20Master%20Windows/badge/icon?subject=%20%20%20%20Windows%20CI%20)](http://jenkins.eprosima.com:8080/view/Micro%20XRCE/job/Micro-CDR%20Nightly%20Master%20Windows/)
+
+### Public API Testing [4.ii]
+
+Each part of the public API has tests, and new additions or changes to the public API require tests before being added.
+The tests aim to cover typical usage and corner cases.
+
+### Coverage [4.iii]
+
+[![Coverage](https://img.shields.io/jenkins/coverage/cobertura?jobUrl=http%3A%2F%2Fjenkins.eprosima.com%3A8080%2Fview%2FMicro%2520XRCE%2Fjob%2FMicro-CDR%2520Nightly%2520Master%2520Linux%2F)](http://jenkins.eprosima.com:8080/view/Micro%20XRCE/job/Micro-CDR%20Nightly%20Master%20Linux/)
+*eProsima Micro CDR* aims to provide a line coverage **above 90%**.
+*Micro CDR* code coverage policy comprises:
+1. All contributions to *Micro CDR* must increase (or at least keep) current line coverage.
+   This is done to ensure that the **90%** line coverage goal is eventually met.
+1. Line coverage regressions are only permitted if properly justified and accepted by maintainers.
+1. If the CI system reports a coverage regression after a pull request has been merged, the maintainers must study the case and decide how to proceed, mostly reverting the changes and asking for a more thorough testing of the committed changes.
+2. This policy is enforced through the [nightly Micro CDR CI job](http://jenkins.eprosima.com:8080/view/Micro%20XRCE/job/Micro-CDR%20Nightly%20Master%20Linux/).
+
+As stated in [CONTRIBUTING.md](CONTRIBUTING.md), developers and contributors are asked to run a line coverage assessment locally before submitting a PR.
+
+### Performance [4.iv]
+
+**eProsima Micro CDR** does not provide performance tests.
+However, performance is indirectly tested by *eprosima Micro XRCE-DDS Client*.
+
+Any performance regression detected in *eprosima Micro XRCE-DDS Client* would be analyzed and, in case that it is related to **eProsima Micro CDR** or could be solved modifying this library, changes could be made to the library in order to revert the performance regression.
```

Comment:
```suggestion
Any performance regression detected in *eProsima Micro XRCE-DDS Client* would be analyz¡sed and, in case it is related to **eProsima Micro CDR** or could be solved by modifying this library, changes can be made to the library in order to revert the performance regression.
```